app/(app)/feedback.tsx

```diff
@@ -1,20 +1,124 @@
-import React from 'react';
+import { useTheme } from '@react-navigation/native';
+import { useRouter } from 'expo-router';
+import React, { useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { TextInput } from 'react-native-gesture-handler';
+import FeedBackBanner from '../../assets/artworks/FeedBackBanner';
+import BackNavigationButton from '../../components/buttons/BackNavigationButton';
+import FeedBackRatingButton from '../../components/buttons/FeedBackRatingButton';
+import SubmitFeedbackButton from '../../components/buttons/SubmitFeedbackButton';
+import Space from '../../components/common/Space';
 import StyledText from '../../components/common/StyledText';
 import MainContainer from '../../components/container/MainContainer';
+import FeedbackSentModal from '../../components/modals/FeedbackSentModal';
+import { typography } from '../../config/typography';
 
-// TODO: implement feedback page
-/**
- * TASKS:
- * Implement feedback page
- * Should include a reactions component, a textarea and button
- */
+export type TypeRatingStates = {
+  icon: string;
+  text: string;
+  value: number;
+};
+
+const Feedback = () => {
+  const { colors } = useTheme();
+  const router = useRouter();
+  const [showModal, setShowModal] = useState(false);
+  const [selectedRating, setSelectedRating] = useState(2);
+  const [description, setDescription] = useState('');
+  const ratingStates: Array<TypeRatingStates> = [
+    { icon: '😔', text: 'Bad', value: 0 },
+    { icon: '😐', text: 'Okay', value: 1 },
+    { icon: '😊', text: 'Great', value: 2 },
+  ];
+  const openModal = () => {
+    setShowModal(true);
+  };
 
-const feedback = () => {
   return (
-    <MainContainer preset="scroll">
-      <StyledText>feedback</StyledText>
+    <MainContainer preset="scroll" safeAreaEdges={['top']}>
+      <FeedbackSentModal showModal={showModal} />
```

Comment:
You do not need this. you could just use the usual header component in the Stack screen, set the background as transparent. then inside the main container use react native's `ImageBackground` component to render the banner

something like this 

```
<Stack.Screen
  ...
  options={{
    ...
    headerTintColor: '#ecf1f1',
    headerStyle: {
      backgroundColor: 'rgba(0,0,0,0.4)',
          },
        }}
   />

  <ImageBackground
        source={{ uri: <url for banner> }}
        style={[styles.banner, { height: height / 4 }]}
        resizeMode='cover'
   />
```

Reply:
I second this

Reply:
@kharioki @brianwachira  is this ae051f7859d1cadccfd054bc4ee2096a614f250d what you expect ? I've implemented the feedbackBanner as an SVG component since it is smaller in size compared to a PNG of the same image, it's also sharper and allows for dynamic styling.

Reply:
I think you did not get what I was trying to communicate. you do not need to create the banner component. you can add the banner svg to the artworks folder where the other svgs are located. then use an Image background so that when you set the headerstyle to translucent it has the image in its background as seen in the design

Reply:
I have fixed it